website/docs/docs/use-dbt-semantic-layer/tableau.md

```diff
@@ -16,11 +16,12 @@ This integration provides a live connection to the dbt Semantic Layer through Ta
 
 ## Prerequisites
 
-1. You must have [Tableau Desktop](https://www.tableau.com/en-gb/products/desktop) installed with version 2021.1 or greater
+- You have [configured the dbt Semantic Layer](/docs/use-dbt-semantic-layer/setup-sl) and are using dbt v1.6 or higher.
```

Comment:
We should come to a consensus as a team on how to handle versioning. Here we say v1.6 or higher but below we say 1.6 or earlier. We should choose either `earlier and later` or `higher and lower` I don't have a preference either way but we should stay consistent. 
```suggestion
- You have [configured the dbt Semantic Layer](/docs/use-dbt-semantic-layer/setup-sl) and are using dbt v1.6 or higher.
```

Reply:
i tend to use higher and lower usually